sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -480,6 +487,82 @@ def test_with_input_types_decorator_violation(self):
         pc = p | Create(l_3_tuple)
         _ = pc | beam.CombineGlobally(self.fn)
 
+#
+# Test cases for streaming.
+#
+@attr('ValidatesRunner')
+class TimestampCombinerTest(unittest.TestCase):
+
+  class RecordFn(beam.DoFn):
+    def process(self, elm=beam.DoFn.ElementParam, ts=beam.DoFn.TimestampParam):
+      yield (elm, ts)
+
+  @unittest.skip('BEAM-8657')
+  def test_combiner_earliest(self):
+    """Test TimestampCombiner with EARLIEST."""
+    options = PipelineOptions()
+    options.view_as(StandardOptions).streaming = True
+    p = TestPipeline(options=options)
+
+    main_stream = (p
+                   | 'main TestStream' >> TestStream()
+                   .add_elements([window.TimestampedValue(('k', 100), 0)])
```

Comment:
I'd probably be a stronger test if the timestamps weren't the endpoints, e.g. 2 and 7. (Similarly below.)

Reply:
Updated to be 2 and 7 respectively.